pkg/api/pod/util.go

```diff
@@ -55,6 +55,30 @@ func VisitContainers(podSpec *api.PodSpec, visitor ContainerVisitor) bool {
 	return true
 }
 
+// VisitContainersUnconditional is like VisitContainers, but it ignores feature gates for
```

Comment:
Would you mind adding some comments around when a developer should use `VisitContainersUnconditional` and when they use `VisitContainers`? Particularly I think it'd be helpful to make it clear to the developer what the "default" should be.

Reply:
I thought I did that already, but now that you mention it we can make it even clearer by making this `visitContainersUnconditional`